### PR TITLE
Add CRUD operations

### PR DESCRIPTION
**Pull request by pandermatt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,40 @@ render(Dato::Live.new(MyComponent, my_query, preview: true, live: true))
...
+```ruby
+Dato::Client.new.items.all(item_type_id: '456')
+```
+
+### Find
+
+```ruby
+Dato::Client.new.items.find(item_id: '123')
+```
+
```
Comment (coorasse) [inline README.md]: let's sort them `all`, `find`, `create`, `update`, `destroy`.

File: fixtures/vcr_cassettes/Dato_Client/can_execute_a_query_without_errors.yml
Comment (coorasse) [inline fixtures/vcr_cassettes/Dato_Client/can_execute_a_query_without_errors.yml:1]: we don't need to recreate this cassette, right? can you roll it back if not needed?

File: fixtures/vcr_cassettes/Dato_PublishController/expires_all_caches.yml
Comment (coorasse) [inline fixtures/vcr_cassettes/Dato_PublishController/expires_all_caches.yml:1]: also other cassettes that don't need an update, please don't update them

File: spec/dato/client_spec.rb [hunk 2]
```diff
@@ -27,4 +29,35 @@
...
+      response = client.items.all(item_type_id: item_type_id)
+      expect(response.parse["data"].last["attributes"]["title"]).to eq("Hello world")
+      expect(response.code).to eq(200)
+
+      response = client.items.find(item_id: item_id)
+      expect(response.parse["data"]["id"]).to eq(item_id)
+      expect(response.parse["data"]["attributes"]["title"]).to eq("Hello world")
+      expect(response.code).to eq(200)
+
+      response = client.items.update(attributes: {title: "Hello world 2"}, item_id: item_id)
```
Comment (coorasse) [inline spec/dato/client_spec.rb:49]: some spacing is missing here

**Review by coorasse — CHANGES_REQUESTED** [review]
Implementation looks great. Please update the version number and add details in the CHANGELOG file.
Please also run standard because it seems that the formatting is weird sometimes.

File: spec/dato/client_spec.rb (hunk 2 above)
Comment (pandermatt) [reply spec/dato/client_spec.rb:49]: `bundle exec standardrb --fix` is removing the spaces

**Comment by pandermatt** [comment]
> Please also run standard because it seems that the formatting is weird sometimes.

I did 😃

**Review by coorasse — APPROVED** [review]
Thank you @pandermatt

**Pull request merged**